frontend/src/app/constants/constants.d.ts

```diff
@@ -589,7 +589,7 @@ interface PersonUpdate extends Address {
   gender: Gender;
   residentCongregateSetting: boolean | null | undefined;
   employedInHealthcare: boolean | null | undefined;
-  tribalAffiliation: TribalAffiliation;
+  tribalAffiliation: TribalAffiliation | undefined;
```

Comment:
Did this PR get another bug fix mixed into it? What's happening with `tribalAffiliation`?

Reply:
This is actually the same bug. Since we weren't allowing `tribalAffiliation` to be `undefined` (it _should_ be allowed to be undefined), we had been setting it to a default value (`null`) in `PersonForm`. The `ComboBox` component, which is from the React USWDS maintained by Truss, only takes a `string` or `undefined` type as its value. We were therefore assigning its value as `patient.tribalAffiliation || undefined`. The `ComboBox` fires an initial `onChange` event, which we can safely ignore if the changed value is equal to the previous value, but in the case of `tribalAffiliation` the changed value was `undefined` and previous value was `null`. By allowing `tribalAffiliation` to be initialized as `undefined`, we can safely ignore the initial `ComboBox` `onChange` event that was causing the form to be marked as dirty.

Reply:
Got it - thanks!